_posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: The Ready Player One Approach to Defense
+date: 21-07-2016
+categories: vanity security thinks sci-fi
+---
+# The Ready Player One Approach to Defense
+
+> What if computer emulation wasn't just about nostalgia? What if we could leverage the power of being olde and having knowledge of defunct hardware and software and legitimately put it to use today?
+
+![Rad!]({{site.url}}assets/images/retro-console.gif)
```

Comment:
Yeah I agree completely but I think that committing binaries to the repository is a decision that it's difficult to claw back from whereas linking remotely and then updating later is a lot easier to do.

I don't doubt we can automate a git-annex workflow whereby images commited in git-annex are hosted remotely on fuc but the binary data doesn't make its way into the repo so we don't end up versioning a bunch of blobs and making clones slow